docs/src/modules/components/overview/charts/mainDemo/HeatmapDemo.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
+import * as React from 'react';
+import { interpolateBlues } from 'd3-scale-chromatic';
+import { Heatmap } from '@mui/x-charts-pro/Heatmap';
+import Typography from '@mui/material/Typography';
+import { AxisConfig } from '@mui/x-charts/models';
+
+const weekDays = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+const hours = ['6am-10am', '10am-12am', '12am-5pm', '5pm-8pm', '8pm-12pm'];
```

Comment:
In heatmap example in the "Advanced charts" section we use a 24-hour clock, but here we're using a 12-hour one. Should we standardize on one over the other?